scripts/play.ts

```diff
@@ -0,0 +1,550 @@
+
+    if (args.framework) {
+      console.warn(
+        chalk.yellow`The --framework/-f argument is ignored when a playground already exists.`,
+      );
+    }
+  } else {
+    console.log(`Creating a new playground: ${chalk.green(playgroundName)}`);
+
+    await createPlayground(playgroundName, playgroundDir, {
+      framework: args.framework ?? DEFAULT_FRAMEWORK,
+      stage: args.stage,
+      dryRun: args["dry-run"],
+    });
+  }
+
+  if (args.start) {
+    await exec("yarn", ["dev"], {
+      cwd: playgroundDir,
+      env: {
+        CONCURRENTLY_PREFIX: `${chalk.white.dim(playgroundName)} [{name}]`,
+      },
+      stdio: "inherit",
+      dryRun: args["dry-run"],
+    });
+  }
+
+  process.off("uncaughtException", handleUncaughtException);
+}
+
+/**
+ * Creates a new playground.
+ *
+ * @param name - The name of the playground. It must be unique.
+ * @param dir - The directory of the playground. It must be unique.
+ * @param options - Options that determine the function's behavior.
+ */
+async function createPlayground(
+  name: string,
+  dir: URL,
+  options: DryRunOption & {
+    /**
+     * The stage on which the playground is run.
+     *
+     * @defaultValue `"staging"`
+     */
+    stage?: Args["stage"];
+
+    /**
+     * The framework used to bootstrap the playground.
+     *
+     * @defaultValue `"next"`
+     */
+    framework?: Args["framework"];
+  } = {},
+) {
+  switch (options.framework ?? "next") {
+    case "next": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/nextjs-starter-prismic-minimal-ts.git",
+        dir,
+        { dryRun: options.dryRun },
+      );
+
+      await updatePackageJSON(
+        dir,
+        { scripts: { "next:dev": "next dev --port=8001" } },
+        { dryRun: options.dryRun },
+      );
+
+      await updateSliceMachineConfig(
+        dir,
+        { localSliceSimulatorURL: "http://localhost:8001/slice-simulator" },
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    case "nuxt": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/nuxt-starter-prismic-minimal.git",
+        dir,
+        { dryRun: options.dryRun },
+      );
+
+      await updatePackageJSON(
+        dir,
+        { scripts: { "nuxt:dev": "nuxt dev --port=8001" } },
+        { dryRun: options.dryRun },
+      );
+
+      await updateSliceMachineConfig(
+        dir,
+        { localSliceSimulatorURL: "http://localhost:8001/slice-simulator" },
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    case "sveltekit": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/sveltekit-starter-prismic-minimal.git",
+        dir,
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    default: {
+      throw new CommandError(`Unsupported framework: ${options.framework}`);
+    }
+  }
+
+  if (!options.dryRun) {
+    await fs.rm(new URL("./.git", dir), { recursive: true });
```

Comment:
😮 It looks like we don't need to! I thought a Git repo within another would cause issues if it wasn't marked as a submodule, but it seems to work okay. This is better since it lets us track changes and quickly revert a playground.

We still need to remove the `origin` remote since we don't want the starters to be changed.

Do you think we should remove all Git history and initialize a new repo?